src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,45 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Driver petrenkoPP = new Driver("Petrenko P.P.", "KIA000001");
+        Driver ivanenkoII = new Driver("Ivanenko I.I.", "KIB000001");
+        Driver boykoOO = new Driver("Boyko O.O.", "KIC000001");
+        Driver sydorenkoBB = new Driver("Sydorenko B.B.", "KIE000001");
+        driverService.create(petrenkoPP);
+        driverService.create(ivanenkoII);
+        driverService.create(boykoOO);
+        driverService.create(sydorenkoBB);
```

Comment:
you can add them to list and create them in forEach, its proposition in case you have 100 manufacturers/drivers/cars :) but in case of 4 I think its ok :)